<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabProj</class>
 <widget class="QDialog" name="LabProj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1032</width>
    <height>586</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1032</width>
    <height>586</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LabProj</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labSizeLabel">
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Labirint size: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="labSizeField">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="labBuildButton">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Build</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Step by step drawing</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="lab1algList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Backtracker</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wall Follower</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dead End Filler</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="solve1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Solve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Time spent: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab1Time">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Cell %:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cellPer1">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>104</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="lab2AlgList">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>23</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Backtracker</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Wall Follower</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dead End Filler</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="solve2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Solve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Time spent: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab2Time">
         <property name="minimumSize">
          <size>
           <width>59</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Cell %:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cellPer2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGraphicsView" name="labField">
         <property name="minimumSize">
          <size>
           <width>502</width>
           <height>502</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>502</width>
           <height>502</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="labField2">
         <property name="minimumSize">
          <size>
           <width>502</width>
           <height>502</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>502</width>
           <height>502</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
